### DISCUSSION TITLE
v0.69.0

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#5244] [Announcements]
### New Features

- Commenting and mentions
- Database migrations are now run automatically #5141 
- Ability to set a custom theme accent color #4897

### Improvements

- Filtering editor block menu is now better ordered to match search
- New and updated translations
- `Escape` key now defocuses the editor
- Mermaid upgraded to `9.2.0`
- Added support for Nix code highlighting
- Added `curl` into base image for easier docker healthchecks
- `/_health` endpoint now checks database connections
- Selected image can now be zoomed with `Space`
- Document titles now have spaces trimmed on save
- Downloading a file as markdown now gets a `.md` extension
- Import now supports zip files created on Windows
- Image resize operations can now now be undone with `Mod`+`z`
- Added support for query params during OIDC login #5129 
- Added list indent and outdent controls on mobile
- Strikethrough of inline code is now supported
- Added a "success" notice style

### Fixes

- Fixed unable to scroll documents by swiping ontop of images on mobile
- Fixed Google LookerStudio embeds
- Fixed TLDraw embeds #4968
- Fixed embedding of YouTube links with a timestamp #5051
- Fixed pasting from Sheets/Numbers no longer pastes an image instead of a table
- Fixed dollar symbols clashing with math blocks #5080
- Fixed incorrect coloring on links in headings #5126
- Fixed label rendering bug on Mermaid diagrams
- Fixed title becomes uneditable in Firefox #5199
- Fixed expensive double loop potentially locks up server
- Fixed infinite loop when connecting Slack integration
- Editor toolbar is now hidden when dragging content
- `Cmd+a` and `Cmd+e` now works correctly in code blocks on Mac. 
- KaTeX is now correctly parsed in shared documents 
- Inline code is now more readable in dark theme
- Improved sanitization of log messages 
- It's now possible to upload multiple files at once from the editor block menu #4957
- Line numbers in code blocks appear in the wrong position
- `LOG_LEVEL` should now be used to choose the level of logging #4937 
- `profile.name` is no longer mandatory anymore in OIDC provider
- Copying a header link now always points to the non-edit url
- Deleting a collection now returns draft documents

<hr /><em>This discussion was created from the release <a href='https://github.com/outline/outline/releases/tag/v0.69.0'>v0.69.0</a>.</em>

**Comment by bmm-alc** [comment]
Thanks for the work on comments. Something a lot of users were waiting for.

**Comment by DrJume** [comment]
Nice
